content/ja/diversity_sep2020.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: Numpyの多様性とチームの参加に関する声明
+title: NumPyの多様性とチームの参加に関する声明
 sidebar: false
 ---
 
 
-_Natureに掲載されたNumPyの論文が発表された後のソーシャルメディアで議論が行われ、Numpyの多様性とチームへの参加に関する懸念が提起されました。 そこで以下の声明を発表したいと思います_
+_Natureに掲載されたNumPyの論文が発表された後のソーシャルメディアで議論が行われ、NumPyの多様性とチームへの参加に関する懸念が提起されました。 そこで以下の声明を発表したいと思います_
 
 
-私たちがチームに加わりたい人にオープンで平等であるとき、私たちはチームとコミュニティに最善を尽くしていると強く信じることが出来ます。 プロジェクトの発足当初からの私達は国際的なチームでありたいと思っていました。また、私たちは多様なバックグラウンドや専門知識を持つ個人同士のコラボレーションの価値を認識しています。 誰もが歓迎され、助け合い、評価される文化は、Numpy プロジェクトの中核です。
+私たちがチームに加わりたい人にオープンで平等であるとき、私たちはチームとコミュニティに最善を尽くしていると強く信じることが出来ます。 プロジェクトの発足当初からの私たちは国際的なチームでありたいと思っていました。また、私たちは多様なバックグラウンドや専門知識を持つ個人同士のコラボレーションの価値を認識しています。 誰もが歓迎され、助け合い、評価される文化は、NumPy プロジェクトの中核です。
 
 ## 過去
 
-オープンソースへ貢献することは、歴史的に疎外されてきた人達、特に女性や、多くの社会的制約や期待のために参加するためにより多くの障害に直面してきた人達に取って、娯楽であり続けてきました。 オープンソースのプロジェクトでは、大きな多様性の問題があり、レポートとしてもまとめられています。(例えば、 [2017 GitHub Open Source Survey](https://opensourcesurvey.org/2017/) と [こちらのブログ投稿](https://medium.com/tech-diversity-files/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996)).
+オープンソースへの貢献はいつでも、歴史的に疎外されてきた集団、特に女性にとって、虐げられた娯楽であり続けてきました。たくさんの社会的制約や、偏見のためです。オープンソースのプロジェクトでは、重大な多様性の問題があり、文書にもよくまとめられています。(例えば、 [2017 GitHub Open Source Survey](https://opensourcesurvey.org/2017/) と [このブログポスト](https://medium.com/tech-diversity-files/if-you-think-women-in-tech-is-just-a-pipeline-problem-you-haven-t-been-paying-attention-cb7a2073b996)).
```

Comment:
This is the most serious mistranslation I found and @AtsushiSakai you might want to check this. Other mistakes are trivial and not very serious.

Reply:
Should we drop this file translation completely? I removed it from Crowdin already, because the page is not linked from other pages, it's really just kept for historical reasons.

Reply:
Ah, I didn't notice that. Feel free to do so if it's necessary. Rebasing is not difficult.

Reply:
done in gh-430

Reply:
apologies for the extra work that caused, I had excluded it in `crowdin.yml` but missed there was already a copy in `master`